zproject/urls.py

```diff
@@ -300,6 +301,8 @@
     ),
     # realm/deactivate -> zerver.views.deactivate_realm
     rest_path("realm/deactivate", POST=deactivate_realm),
+    # realm/test_custom_welcome_bot_message -> zerver.views.send_test_custom_welcome_bot_message
```

Comment:
`zerver.views.send_test_custom_welcome_bot_message` -- no such file exists

Reply:
My bad, I thought it was a function as the above comment has "deactivate_realm," which is not a file name but a function inside realm.py. I'll correct it. Thanks for pointing it out.